### ISSUE TITLE
Unable to find shader

### ISSUE THREAD

**Issue by TracyLestrade** [xenko3d/xenko#492]
Enviroment: visual studio 2017 with MSBuild 2019. windows 10 x64
first off. I cannot install visual studio plugin. and write a shader seems working on editor. but it occurds an exception about unable to find shader.
should I deal with the plugin problem first?

**Comment by TracyLestrade** [comment]
I'm sure It' not visual studio plugin issue. But I don't really know how it works. 
Is shader compiled to byte code into database? or just use origin shader string, and be compiled before application launching?

**Comment by TracyLestrade** [comment]
Problem solved. My shader file(.xksl) is in **$(Project)/Shader** folder. but in **[XenkoProject].Game.xkpkg** file, configuration [AssetFolders] doesn't exist **Shader** folder, It uses **Effects** folder by default.

**Comment by TracyLestrade** [comment]
why this issue just come to me. Now it makes scene.

**Comment by devjarmo** [comment]
I think this issue should be re-opened @xen2 as I also hit the issue. I created a fresh project and attempted to add a single shader. The Editor renders the shader no problem (in the Project folder), however when compiling and running the game, the shader cannot be found, the [XenkoProjectName].xkpkg was looking in Effects. I created Effects and moved the shader to that folder and now the game compiles.
